components/registry-facade/cmd/run.go

```diff
@@ -150,7 +148,22 @@ var runCmd = &cobra.Command{
 			log.WithError(err).Fatal("cannot create registry")
 		}
 
-		go watchConfig(configPath, reg)
+		ctx, cancel := context.WithCancel(context.Background())
+		defer cancel()
+
+		err = watch.File(ctx, configPath, func() {
+			ctx, cancel := context.WithTimeout(context.Background(), 60*time.Second)
+			defer cancel()
+
+			err := reg.UpdateStaticLayer(ctx, cfg.Registry.StaticLayer)
```

Comment:
I don't see where the config is actually re-read from disk. The prior implementation had a `config.GetConfig(fn)` call to that end.

Reply:
Fixed